apps/templates/password_changed/email.html

```diff
@@ -0,0 +1,16 @@
+
+{% block container %}
+    <p>
+        Your password has been changed successfully.
+        Time:{{time}}
+        Location:{{location}}
+        Device:{{device}}
+        If you are aware of this change, you can disregard this email.
+        If this was not triggered by you, click the button below to reset your password.
+        <a class="button-link"
+        href="{{ protocol }}://{{ domain }}{% url 'user-change_password' version='v2' %}"
```

Comment:
`{% url 'user-change_password' version='v2' %}` where does this point to?

Reply:
Point to change_password as suggested earlier

Reply:
Changed that in  commit `1f3833c`